### DISCUSSION TITLE
Image Base64 Upload

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#13437] [Feature Requests]
Given the scenario, that i dont want all my users to give permission to upload files (in genreral). I want to give permission to upload specific fields.
Therefore i propose the possibility to upload a base64 (image) string to an image field. Directus would there detect the base64 string, converts it back to an image/file.

This would allow a finer granularity in permission handling.

**Comment by br41nslug** [comment]
This idea is already being discussed here #11753 :)

**Reply by NilsBaumgartner1994** [reply]
Implemented it myself in a hook. see discussion abouve for solution
